week1/ubuntu-intro.md

```diff
@@ -0,0 +1,90 @@
+# Intro to Ubuntu
+
+![image alt](https://assets.ubuntu.com/v1/8dd99b80-ubuntu-logo14.png)
+
+## What ?
+
+- Ubuntu is a free and open-source Linux distribution.
+> **What is linux ?**
+> Linux is an open-source operating system and a low-cost or free alternative to Microsoft Windows
+
+> **open-source** meaning that everone can view the code of Ubuntu, and anyone can contribute to it's progress.
+
+## Why ?
+
+- It's lightweight and high performant, unlike windows.
+- It's free, and it also comes with a massive library of tools that are also free to download.
+- Better security and privacy
+> Linux ensures the privacy of user’s data as it never collects much data from the user while using its distributions or software but this is not true for many other operating systems.
+- Gives you deeper control over you computer, since it's lower level and does not hide things from you. 
+
+
+## How ?
+
+
+### Let's install it together :tada: 
+
+Prerequisites:
+
+1) Have a usb with Ubuntu image on it.
+2) Make sure to have a computer :laughing: :stuck_out_tongue_winking_eye:, and enough storage for atleast 20 GB.
+> Make sure to help others if you have finished the setting up/ or already have Ubuntu. :pray: 
+
+
+
+
+### Step 1
+Go to ` Control Panel > System and Security > Create and format hard disk partitions`
+
+![image alt](https://i.imgur.com/OUhvP5T.gif)
+
+### Step 2
+
+- Choose which disk you want to partition.
+> Partitioning is the process of dividing a disk into one or more regions, the so called partitions. partitioning a hard disk is helpful for either better organization or for having 2 operating systems on the same hard disk. 
+
+- And then shrink the volume (hard disk) to 20 - 40 GB.
+
```

Comment:
I'd suggest that the students review that they've done Step 2 correctly with a mentor before continuing. If they get this step wrong, they could potentially corrupt their hard-drive.